tlatools/org.lamport.tlatools/src/tlc2/output/MP.java

```diff
@@ -1296,6 +1296,31 @@ else if (parameters.length == 2) {
                     + "\nbut TLC can only handle behaviors of length up to 65535 states. The last\n"
                     + "state in the behavior is:\n%1%");
         	break;
+        case EC.TLC_CONFIG_PROPERTY_ACTION_LEVEL_AND_STUTTERING_SENSITIVE:
+            b.append("The formula %1% is an action-level formula (i.e., it contains no temporal operators), "
+            		+ "but it was used as a PROPERTY, where a temporal formula is normally required. "
```

Comment:
Why "normally"? To me this sounds like in certain cases other kinds of formulas might be acceptable. (Remember that any state formula is also a temporal formula.) I suggest simply removing "normally".